src/ZoneServer/World/Actors/Components/StateLockComponent.cs

```diff
@@ -50,6 +50,7 @@ public StateLockComponent(IActor owner)
 
 			this.RegisterState(new(StateType.Stunned, [LockType.Movement, LockType.Attack]));
 			this.RegisterState(new(StateType.KnockedBack, [LockType.Movement, LockType.Attack]));
+			this.RegisterState(new(StateType.KnockedDown, [LockType.Movement, LockType.Attack]));
```

Comment:
There was no `KnockDown` lock state yet because knock backs and knock downs are conceptually the same in terms of locks in most games. Did you find a reason to differentiate them here?

Reply:
Alright, good to know!